src/main/java/org/mule/extension/ftp/internal/command/FtpWriteCommand.java

```diff
@@ -159,8 +160,23 @@ private void closeSilently(Closeable closeable) {
     }
   }
 
+  /**
+   * Method to remove '/' when the file path is under root directory only.
+   * ex. '/abc.txt' 
+   * @param path, normalized path 
+   * @return updated path
+   */
+  private String removeSlashUnderRootDirOnly(String path) {
+    if (path != null && path.length() > 1 && path.charAt(0) == SEPARATOR.charAt(0)
```

Comment:
Hey @rogerwang1102  you can use here StringUtils.isEmpty here

Reply:
sorry is isBlank

Reply:
But that is checking empty, but I want the length to be more than 1.